promitor-agent-resource-discovery/values.yaml

```diff
@@ -155,10 +157,29 @@ rbac:
     name: promitor-resource-discovery
     annotations: {}
 
+    ## Set this to true if you plan on using Pod Security Policy
+    automountServiceAccountToken: false
+
 nodeSelector: {}
 
+## securityContext and containerSecurityContext are using secure defaults.
+## Only override if you have a good reason to.
 securityContext:
-  enabled: false
+  enabled: true
+  runAsGroup: 10000
+  runAsNonRoot: true
+  runAsUser: 10000
+  seccompProfile:
+    type: RuntimeDefault
+
+containerSecurityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+    - ALL
+  enabled: true
```

Comment:
Are we sure this does not break things?

Reply:
I have tested that everything still works on my end. We also know that these settings are working fine for the scraper agent.